kartothek/io/testing/read.py

```diff
@@ -726,3 +728,35 @@ def test_binary_column_metadata(store_factory, bound_load_dataframes):
 
     # Assert column names are of type `str`, instead of `bytes` objects
     assert set(df.columns.map(type)) == {str}
+
+
+@pytest.mark.xfail(
+    LooseVersion(pa.__version__) < "0.16.1.dev308",
```

Comment:
I guess this is the version pointing to the master commit? (i.e. can we merge?)

Reply:
This is the nightly master build of Sunday night.

Reply:
Yes, we can merge.